### PR TITLE
PE-1198

### PR DESCRIPTION
**Pull request by gbalabasquer**
# Description

# Contribution Checklist

- [ ] PR title starts with `(PE-<TICKET_NUMBER>)`
- [ ] Code approved
- [ ] Tests approved
- [ ] CI Tests pass

# Checklist

- [ ] Every contract variable/method declared as public/external private/internal
- [ ] Consider if this PR needs the `officeHours` modifier override
- [ ] Verify expiration (`30 days` unless otherwise specified)
- [ ] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [ ] Validate all addresses used are in Goerli changelog or known
- [ ] Notify any external teams affected by the spell so they have the opportunity to review
- [ ] Deploy spell to Goerli `ETH_GAS_LIMIT="XXX" ETH_GAS_PRICE="YYY" make deploy`
- [ ] Ensure contract is verified on `Goerli` etherscan
- [ ] Change test to use Goerli spell address and deploy timestamp
- [ ] Cast spell on Goerli `make spell="0x-deployed-spell-address" cast-spell`
- [ ] Run `make archive-spell` or `make date="YYYY-MM-DD" archive-spell` to make an archive directory and copy `DssSpell.sol`, `DssSpell.t.sol` and `DssSpell.t.base.sol`
- [ ] `squash and merge` this PR

### CODE REVIEW

File: src/DssSpell.sol [hunk 1]
```diff
@@ -40,56 +43,34 @@ contract DssSpellAction is DssAction {
...
+
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmXGgakY#poll-detail
+        DssExecLib.setIlkStabilityFee("ETH-C", ZERO_SEVENTY_FIVE_PCT_RATE, true);
+        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_SEVENTY_FIVE_PCT_RATE, true);
+        DssExecLib.setIlkStabilityFee("WBTC-C", ONE_PCT_RATE, true);
+        DssExecLib.setIlkStabilityFee("YFI-A", ONE_FIVE_PCT_RATE, true);
+        DssExecLib.setIlkAutoLineDebtCeiling("RETH-A", 20 * MILLION);
+        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A", 4 * MILLION);
+        // DssExecLib.setIlkAutoLineDebtCeiling("DIRECT-COMPV2-DAI", 70 * MILLION); ilk does not exist in Goerli
```
Comment (kmbarry1) [inline src/DssSpell.sol:73]: Should there also be a commented-out line for the AAVE D3M debt ceiling raise? From the linked poll:

* Increase the **Aave v2 D3M** Maximum Debt Ceiling by 15 million DAI from 5 million DAI to **20 million DAI**.

File: src/DssSpell.sol [hunk 2]
```diff
@@ -40,56 +43,34 @@ contract DssSpellAction is DssAction {
...
-        // Note: only on mainnet
-
-        // Bump changelog
-        DssExecLib.setChangelogVersion("1.14.9");
+    function actions() public override {
+        // Stablecoin vault offboarding
+        // https://vote.makerdao.com/polling/QmemXoCi#poll-detail
+        Fileable(MCD_SPOT).file("USDC-A",   "mat", 15 * RAY); // 1500% collateralization ratio
+        Fileable(MCD_SPOT).file("PAXUSD-A", "mat", 15 * RAY);
+        Fileable(MCD_SPOT).file("GUSD-A",   "mat", 15 * RAY);
```
Comment (kmbarry1) [inline src/DssSpell.sol:60]: Going to check explicitly later but I'm assuming the other parameters mentioned in the poll already have the specified values?

File: src/DssSpell.sol (hunk 1 above)
Comment (kmbarry1) [reply src/DssSpell.sol:73]: Ah I see now the discussion in the new-spells channel, never mind.

File: src/DssSpell.sol (hunk 2 above)
Comment (kmbarry1) [reply src/DssSpell.sol:60]: Confirmed, `line`, `chop`, and `tip` all have the values specified in the poll already, at least on Goerli.

**Comment by kmbarry1** [comment]
Review complete. All tests pass for me except for `testAuth` which errored out due to an RPC failure. I'll try to run the tests again tomorrow but IMO it would be fine to proceed with Goerli deployment.

**Comment by gbalabasquer** [comment]
> Review complete. All tests pass for me except for `testAuth` which errored out due to an RPC failure. I'll try to run the tests again tomorrow but IMO it would be fine to proceed with Goerli deployment.

Yeah passing the tests locally is sometimes complex. CI is passing so I think for Goerli we are more than fine.

File: src/DssSpell.sol [hunk 3]
```diff
@@ -40,56 +43,34 @@ contract DssSpellAction is DssAction {
...
-        //Increase WSTETH-B Stability Fee to 0.5%
-        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_FIVE_PCT_RATE, true);
+    uint256 constant ZERO_SEVENTY_FIVE_PCT_RATE = 1000000000236936036262880196;
+    uint256 constant ONE_PCT_RATE               = 1000000000315522921573372069;
+    uint256 constant ONE_FIVE_PCT_RATE          = 1000000000472114805215157978;
 
-        // Reduce RETH-A Stability Fee to 0.5%
-        DssExecLib.setIlkStabilityFee("RETH-A", ZERO_FIVE_PCT_RATE, true);
+    uint256 constant MILLION = 10 ** 6;
+    uint256 constant RAY     = 10 ** 27;
```
[the remark this answers is not in the document]
Comment (gbalabasquer) [reply src/DssSpell.sol:51]: Let's leave it as it is.

File: src/DssSpell.sol [hunk 4]
```diff
@@ -40,56 +43,34 @@ contract DssSpellAction is DssAction {
...
-
-        // ---- SF-001 Contributor Vesting ----
-        // Note: only on mainnet
-
-        // Bump changelog
-        DssExecLib.setChangelogVersion("1.14.9");
+    function actions() public override {
+        // Stablecoin vault offboarding
+        // https://vote.makerdao.com/polling/QmemXoCi#poll-detail
+        Fileable(MCD_SPOT).file("USDC-A",   "mat", 15 * RAY); // 1500% collateralization ratio
```
[the remark this answers is not in the document]
Comment (gbalabasquer) [reply src/DssSpell.sol]: Changed the DssExecLib thing but why do you want to call storage multiple times to get the same spotter address?

**Review by kmbarry1 — DISMISSED** [review]
LGTM

Tests pass against deployed version.

File: src/test/config.sol [hunk 5]
```diff
@@ -95,9 +95,9 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0x230BFbDbBbA8F07282A67ed4868aC261F23272F2),     // populate with deployed spell if deployed
-            deployed_spell_created:         1676905116,              // use `./scripts/get-created-timestamp.sh <deployment-tx>`
-            deployed_spell_block:           8525393,        // populate with the block where the spell was deployed
+            deployed_spell:                 address(0x5Cc537ed3C9A286128f14376553476dCD0F3C5AB),     // populate with deployed spell if deployed
+            deployed_spell_created:         1677847836,              // use `./scripts/get-created-timestamp.sh <deployment-tx>`
```
Comment (kmbarry1) [inline src/test/config.sol:99]: btw the comment here seems to be outdated, looks like now we use `scripts/time.sh` with the formatted timestamp as the argument

Comment (gbalabasquer) [reply src/test/config.sol]: Comment was outdated but it is not that script either. I have fixed it.

**Comment by gbalabasquer** [comment]
@kmbarry1 @megozu spell has been cast, I've fixed a comment in `config.sol` and archive it.
Ready for final approval.

**Pull request merged**